src/Common/HttpLayer.php

```diff
@@ -153,7 +153,7 @@ protected function buildPlugins(): array
         $contentTypePlugin = new ContentTypePlugin();
 
         $headerDefaultsPlugin = new HeaderDefaultsPlugin([
-            'User-Agent' => 'mailersend-php/'.Constants::SDK_VERSION
+            'UserParams-Agent' => 'mailersend-php/'.Constants::SDK_VERSION
```

Comment:
Maybe this was a find and replace to the whole project that was not intended?

Reply:
oh, I refactored `User` class  to `UserParams` ......fixing